routes/shopifyAuth.js

```diff
@@ -14,24 +14,26 @@ module.exports = function createShopifyAuthRouter({
 
   // This function initializes the Shopify OAuth Process
   router.get('/', function(request, response) {
-    const { query, session } = request;
+    const { query } = request;
     const { shop } = query;
 
-    if (shop) {
-      session.shop = shop;
-      const redirectTo = `https://${shop}/admin/oauth/authorize`;
-      const redirectParams = `?client_id=${apiKey}&scope=${scope}&redirect_uri=${host}/auth/shopify/callback`;
-      response.send(
-        `<!DOCTYPE html>
-        <html>
-          <head>
-            <script type="text/javascript">
-              window.top.location.href = "${redirectTo}${redirectParams}"
-            </script>
-          </head>
-        </html>`
-      );
+    if (shop == null) {
```

Comment:
before we would just do nothing and let the request timeout if there was no shop